docs/source/tracking.rst

```diff
@@ -250,17 +250,19 @@ You run an MLflow tracking server using ``mlflow server``.  An example configura
 Storage
 -------
 
-An MLflow tracking server has two properties related to how data is stored: file store and artifact store.
+An MLflow tracking server has two properties related to how data is stored: a backend store and
+an artifact store.
 
 The *backend store* (exposed as ``--backend-store-uri``) is where the *server* stores run and
 experiment metadata. For backward compatibility, ``--file-store`` option is an alias to this
-option. This can be a local path **file store** specified as ``./path_to_store`` or
-``file://path_to_store``, or a SQL connection string for a *Database backed store*. For the
-latter, argument is expected to be a SQL connection string specified as
+option. This can be a local path *file store* specified as ``./path_to_store`` or
+``file://path_to_store``, or a SQL connection string for a *database backed store*. For the
```

Comment:
Nit: Two slashes should only be used in a `file:` URI when referring to a file on a remote host. We should either use a single slash to refer to a *relative* path as `file:/<relative_path>` or three slashes to refer to an absolute path, for example: `file:///Users/mani/mlflow_store_roots/store1`.

For reference, see the "How many slashes" section in the following article: https://en.wikipedia.org/wiki/File_URI_scheme, as well as https://github.com/mlflow/mlflow/issues/971

Reply:
Great catch. Changed to `file:/path_to_store`--that is how it is used elsewhere in this file.